cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-s2i-webhook-cicd.adoc

```diff
@@ -0,0 +1,98 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-s2i-webhook-cicd"]
+= Tutorial: Using Source-to-Image (S2I) webhooks for automated deployment
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-s2i-webhook-cicd
+:source-highlighter: coderay
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 05-07-2024
+
+Automatically trigger a build and deploy anytime you change the source code by using a webhook. For more information about this process, see link:https://docs.openshift.com/container-platform/latest/cicd/builds/triggering-builds-build-hooks.html[Triggering Builds].
+
+.Procedure
+
+. In your terminal, run the following command to get the GitHub webhook trigger secret:
```

Comment:
Give the why before the how: "To get the GitHub webhook trigger secret, in your terminal run the following command:"